.github/workflows/hotfix-release.yml

```diff
@@ -0,0 +1,148 @@
+name: Hotfix-release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "version to update"
+        required: true
+      prerelease:
+        required: true
+        type: boolean
+        description: Is this a pre-release
+
+jobs:
+  update-tag:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          ref: develop
+      - name: setup go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      - name: setup git
+        run: |
+          git config user.name "Github Actions"
+          git config user.email "info@netmaker.io"
+      - name: create release branch
+        run: |
+          git switch -c release_${{ github.event.inputs.version }}
```

Comment:
`-c` flag not required since the release branch will already be present in a hotfix scenario